releasenotes/notes/DCA-flare-manifests-and-helm-values-0e7056fe726f45d6.yaml

```diff
@@ -0,0 +1,11 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
+  - |
+    Cluster Agent: User config, cluster Agent deployment and node Agent daemonset manifests are now added to the flare archive, when the Cluster Agent is deployed with Helm (version 3.23.0+).
```

Comment:
Could we deploy the latest version of the DCA with a helm chart prior of `3.23.0` ? If so, would the new flare code of the DCA still add the new info ?

Reply:
@hush-hush you could deploy indeed the latest DCA, but without the necessary RBACs/env vars coming from `3.23.0`, you would not get the new info and would get errors similar to the one below in `flare_creation.log` (from using [AddFileFromFunc](https://github.com/DataDog/datadog-agent/blob/main/comp/core/flare/helpers/helpers.go#L58)) :
```
error collecting data from callback for 'agent-daemonset.yaml': Can't collect the Agent Daemonset name and/or namespace from the environment variables AGENT_DAEMONSET and DD_KUBE_RESOURCES_NAMESPACE
error collecting data from callback for 'cluster-agent-deployment.yaml': Can't collect the Cluster Agent Deployment name and/or namespace from the environment variables CLUSTER_AGENT_DEPLOYMENT and DD_KUBE_RESOURCES_NAMESPACE
error collecting data from callback for 'helm-values.yaml': Can't collect the Datadog Helm chart release name and/or namespace from the environment variables CHART_RELEASE_NAME and DD_KUBE_RESOURCES_NAMESPACE
```

Will push changes with `AddFileFromFunc` when GH works again